composer.json

```diff
@@ -34,7 +34,8 @@
         "symfony/console":  "~3.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~5.0"
+        "phpunit/phpunit":    "~5.0",
+        "mikey179/vfsStream": "^1.6"
```

Comment:
Wouldn't go this far IMO - most of this package just uses reads, not writes, so `vfsStream` is probably overkill, since we don't expect writes to ever happen.